components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -153,8 +154,60 @@ func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, de
 		logger.Errorf("cannot create new operation: %s", err)
 		return domain.ProvisionedServiceSpec{}, errors.New("cannot create new operation")
 	}
+
```

Comment:
let's reorganize this block of code by extracting to separate method or something like this (let's discuss)